app/services/services.rb

```diff
@@ -46,15 +46,21 @@ def self.is_assigned_to_cis?(service, cis_id)
   end
 
   def self.service_admins_name_options
-    Admin.service_admins_sorted_by_name.pluck(:name)
+    Admin
+        .service_admins_sorted_by_name
+        .pluck(:id, :name, :surname, :second_surname)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.